src/commands/ml-models-publish.ts

```diff
@@ -0,0 +1,10 @@
+import { Command } from "../command";
+import { FirebaseModel } from "../ml/models";
+
+module.exports = new Command("ml:models:publish <modelId>")
```

Comment:
prefer `export default` in this case
```suggestion
export default new Command("ml:models:publish <modelId>")
```